examples/form/headless/saveAndContinue/src/pages/posts/create.tsx

```diff
@@ -0,0 +1,145 @@
+import { useState } from "react";
```

Comment:
```suggestion
import React, { useState } from "react";
```

same changes as the ones in the docs.